src/test/java/org/scijava/ui/DefaultUIServiceTest.java

```diff
@@ -0,0 +1,32 @@
+
+package org.scijava.ui;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Mockito.mock;
+
+import org.junit.Test;
+import org.scijava.ui.swing.sdi.SwingSDIUI;
+
+/**
+ * Tests for {@link DefaultUIService}
```

Comment:
By convention, I have named the service tests after the interfaces. While technically maybe less correct, it produces shorter class names, and it's not like we unit test any _other_ service interface implementations.

So for consistency, let's call this `UIServiceTest` instead of `DefaultUIServiceTest`.